client/assets/content_bundles/symptom_poster.en.yaml

```diff
@@ -1,39 +0,0 @@
-schema_version: 1
```

Comment:
also new schema version so old apps that see this will put up a force update message

Reply:
The show upgrade dialog doesn't seem to be hooked up... I think it may have gotten lost at some point.